src/docfx/docset/Document.cs

```diff
@@ -20,6 +20,16 @@ internal class Document
         /// </summary>
         public ContentType ContentType { get; }
 
+        /// <summary>
+        /// Gets the MIME type specifed in YAML header or JSON $schema.
+        /// </summary>
+        public string Mime { get; }
```

Comment:
>Mime [](start = 22, length = 4)

if it's equals to Schema, we don't need it?

Reply:
Agree, seems part of Schema?

---
In reply to: [212894056](https://github.com/dotnet/docfx/pull/3291#discussion_r212894056) [](ancestors = 212894056)

Reply:
There is a temporary hack to fallback to `documentType` when YamlMime is `YamlDocument`. The error reporting needs to know the correct mime type, we cannot parse YAML upfront as it is too heavy, I'll leave a TODO